packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -5,72 +5,80 @@ title: Role Assignment
+| `ORG_ADMIN`                             | Organizational administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-only administrator             |                                       |
+| `REPORT_ADMIN`                          | Report administrator                |                                       |
+| `SUPER_ADMIN`                           | Super administrator                 |                                       |
+| `USER_ADMIN`                            | Group administrator                 | [Groups](/docs/reference/api/groups/) |
 
 ### IAM-based standard role Types
 <ApiLifecycle access="ea" />
-We also support the following IAM-based standard roles:
+
+Okta also supports the following IAM-based standard roles:
 | Role type                                                  | Label                               | Permissions                                   |
 | :--------------------------------------------------------- | :---------------------------------- | :-------------------------------------------- |
-| `ACCESS_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | Access Certifications Administrator | `okta.governance.accessCertifications.manage` |
-| `ACCESS_REQUESTS_ADMIN`       <ApiLifecycle access="ea" /> | Access Requests Administrator       | `okta.governance.accessRequests.manage`       |
-IAM-based standard roles could be assigned using [standard](#standard-role-assignment-steps) or [custom role](#custom-role-assignment) assignment operations.
-Similar to other standard roles, these roles are also immutable and can't be updated or deleted.
+| `ACCESS_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | Access certifications administrator | `okta.governance.accessCertifications.manage` |
+| `ACCESS_REQUESTS_ADMIN`       <ApiLifecycle access="ea" /> | Access requests administrator       | `okta.governance.accessRequests.manage`       |
+
+You can assign IAM-based standard roles using [standard](#standard-role-assignment-steps) or [custom role](#custom-role-assignment) assignment operations. These roles are immutable and can't be updated or deleted.
 
 ### Standard Role Assignment steps
-Perform standard role assignment in two steps:
 
-1. Assign a role to a user or group. At this point, the admin has the supported privileges of the role over all resources across organization.
+Perform a standard role assignment in two steps:
+
+1. Assign a role to a user or group. At this point, the admin has the supported privileges of the role over all resources across the organization.
 2. (Optional) If the role supports targets, use one of the [target operations](/docs/reference/api/roles/#role-target-operations) to indicate which specific resource the admin can manage.
 
-Note that the entities involved in standard role assignment are:
+The entities involved in a standard role assignment are:
```

Comment:
```suggestion
These are all of the entities that are involved in a standard role assignment:
```

Reply:
I forget if we translate dev.okta.com, but our translators hate incomplete stem sentences.